main.py

```diff
@@ -75,6 +76,7 @@ def ctran_info():
 # Private Functions
 
 def _create_instances(read_env_data):
+    config.load()
```

Comment:
ingest_env() should only be called if read_env_data is true following Sawyer's code in main.py.

config.load() is currently the only way to set the filename of the config file from the CLI if that is required at some point but if we think that isn't necessary it can be moved to the constructor.

Reply:
Ok sure, I agree with your config.load point. Because config is a singleton it needs to be initialized in its package so that other classes can access it, so you can't initialize it in main. Which makes .load() the de facto __init__ class for config. 
I think you should move .ingest_env inside of .load() then. Move Sawyer's code regarding read_env_data into your class, it belongs there. You can simply pass read_env_data into .load() and use it as needed.

Reply:
That seems fine, I'll do that